dev-util/weka/weka-3.8.6-r1.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source"
+
+inherit desktop java-pkg-2 java-pkg-simple xdg
+
+DESCRIPTION="A Java data mining package"
+HOMEPAGE="https://ml.cms.waikato.ac.nz/weka"
+SRC_URI="https://downloads.sourceforge.net/project/weka/weka-3-8/${PV}/weka-${PV//./-}.zip"
+S="${WORKDIR}/${P//./-}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="app-arch/unzip"
+CP_DEPEND="dev-java/javacup:0"
+DEPEND="${CP_DEPEND}
+	virtual/jdk:1.8"
+RDEPEND="${CP_DEPEND}
+	>=virtual/jre-1.8:*
+	dev-java/commons-compress:0
+	dev-java/hamcrest-core:0
+	dev-java/istack-commons-runtime:0
+	dev-java/jakarta-activation-api:2
```

Comment:
and here, didn't they bundle `jakarta-activation-api-1.2.2`?

Reply:
Yes they do on both I try the latest version and they did work, but if you prefer I can tag to the specify version.

(Also some specify version are not available in the tree that why I did try the lastest to see it all ok)

Reply:
> Yes they do on both I try the latest version and they did work, but if you prefer I can tag to the specify version.
> 
> (Also some specify version are not available in the tree that why I did try the lastest to see it all ok)

I'll leave it to the dev approving this PR.